tests/Structure.php

```diff
@@ -14,6 +14,11 @@ class Structure extends Expression
         'drop'   => 'drop table if exists {table}',
     ];
 
+    public function setEscapeChar($char)
+    {
+        $this->escape_char = $char;
+    }
+
```

Comment:
this class is actually defunct and should me removed in favour of atk4/schema. I'll add a separate ticket for that.

Reply:
I saw the notes about Structure.php somewhere, but would just tackle one thing at a time. So I went with just getting it running.

Reply:
yes, that's good. I'm on vacation now but if i find some free time i'll add ignores to make your PR work and will merge.